src/components/SwapIcon/SwapIcon.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FC } from 'react';
```

Comment:
I'm not sure why this exists? It's not consistent with the way other Icons are used throughout the app. I can only think it's so you didn't have to repeat the wrapper div when you used it? (in which case it could be a sub-component by all means)..

Maybe I'm missing something here? :shrug: